README.md

```diff
@@ -52,7 +52,7 @@ Example:
                     <plugin>
                         <groupId>software.amazon.lambda.snapstart</groupId>
                         <artifactId>aws-lambda-snapstart-java-rules</artifactId>
-                        <version>0.1.0</version>
+                        <version>0.1.1</version>
```

Comment:
You missed line 28 for Gradle.

Reply:
I completely missed that. Thanks. Updated.